tests/test_vcs.py

```diff
@@ -67,6 +67,17 @@ def test_clone():
     shutil.rmtree(tmp, ignore_errors=True)
 
 
+def test_local_clone():
+    tmp = vcs.clone("https://github.com/copier-org/copier.git")
+    assert tmp
+    assert exists(join(tmp, "README.md"))
+
+    local_tmp = vcs.clone(str(tmp))
+    assert local_tmp
+    assert exists(join(local_tmp, "README.md"))
+    shutil.rmtree(local_tmp, ignore_errors=True)
```

Comment:
Don't you need to clear `tmp`  too?

Reply:
I don't think so, the last error in CI is caused by paths in the Windows case